spec/controllers/lets_encrypt/verifications_controller_spec.rb

```diff
@@ -12,7 +17,7 @@
 
   context 'has certificate' do
```

Comment:
The test needs to check the default certificate model and the user customizes model work correctly.

Reply:
Added.